product_yahoo_auction_sst/models/deliver_prefecture.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DeliverPrefecture(models.Model):
+    _name = 'deliver.prefecture'
+    _description = 'Delivery Prefecture'
+
+    name = fields.Char(string='Prefecture Name', required=True)
```

Comment:
Shouldn't we extend `res.country.state` instead by adding `is_deliverable` boolean field, perhaps?